modules/virt-general-availability.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * virt/virt-2-4-release-notes.adoc
+
+[id="virt-general-availability_{context}"]
+= General availability
+
+{VirtProductName} is now a fully supported and generally available feature of the OpenShift Container Platform 4.5. Previously known as container-native virtualization (CNV), OpenShift virtualization enables you to bring traditional virtual machines (VMs) into OpenShift where they run with containers, and are managed as native Kubernetes objects. 
```

Comment:
`where they run with containers` should be `where they run alongside containers`

Reply:
@fabiand - this was updated to indicate "alongside containers..." Tagging @aburdenthehand @sbream for the update.